pyproject.toml

```diff
@@ -16,42 +16,34 @@ no_implicit_optional = true
 show_error_codes = true
 warn_unused_ignores = true
 
-[tool.poetry]
+[tool.uv]
+dev-dependencies = [
+    "ipdb>=0.13.13",
+    "ipython>=8.12.3",
+]
+
+[project]
+requires-python = ">=3.8"
 name = "opendatahub-tests"
 version = "0.1.0"
 description = "Tests repository for Open Data Hub (ODH)"
 authors = []
-license = "Apache License 2.0"
+license = "Apache-2.0"
```

Comment:
are u sure aboout `License`

Reply:
I vote for agpl 3

Reply:
why not apache? checking `kserve` and `opendatahub-operator` as examples of repos in the org - they use apache. we should use the same

Reply:
> are u sure aboout `License`

updated the string